<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>theDialogui</class>
 <widget class="QWidget" name="theDialogui">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Youtube Download</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>WELCOME TO THE NEW YOUTUBE DOWNLOADER</string>
   </property>
  </widget>
  <widget class="QLabel" name="urlLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>237</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Url:</string>
   </property>
  </widget>
  <widget class="QLabel" name="saveLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Save in:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downloadButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>88</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>88</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Save in..</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="urlEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="saveEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>350</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>311</width>
     <height>151</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>311</width>
     <height>151</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icnon/icons/icon.jpeg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.py"/>
 </resources>
 <connections/>
</ui>
